test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -992,69 +990,6 @@ package object test extends CompileVariants {
       }
     }
 
-  private[test] def flatMapStream[R, R1 <: R, A, B](
-    stream: ZStream[R, Nothing, Option[A]]
-  )(f: A => ZStream[R1, Nothing, Option[B]])(implicit trace: Trace): ZStream[R1, Nothing, Option[B]] =
-    ZStream
-      .fromChannel(
-        stream
-          .rechunk(1)
-          .channel
-          .concatMapWithCustom[R1, Any, Any, Any, Nothing, Chunk[Either[Boolean, B]], Any, Any](as =>
```

Comment:
I think this was the only use of this customizable `concatMap`. Maybe we don't have to keep it?

Reply:
Yes I think we can delete it.